ingest/src/commands/pages.ts

```diff
@@ -1,15 +1,23 @@
 import { CommandModule } from "yargs";
 import { updatePages } from "../updatePages";
+import { makeSnootyDataSource } from "../SnootyDataSource";
 
 // TODO: Option for which data source
 type PagesCommandArgs = Record<string, never>;
 
 const commandModule: CommandModule<PagesCommandArgs> = {
   command: "pages",
   handler: async (args) => {
+    const snootySource = await makeSnootyDataSource({
+      baseUrl: "https://mongodb.com/docs/",
+      manifestUrl:
+        "https://snooty-data-api.docs.prod.mongodb.com/projects/docs/master/documents",
```

Comment:
when i go to this URL (even on VPN), it redirects me to https://mongodb.com. what's the expected behavior?

Reply:
You tell me - I got this URL from you! 😜

Reply:
The other day this was actually loading the entire manifest